spirit/comment/flag/admin/templates/spirit/comment/flag/admin/detail.html

```diff
@@ -51,7 +51,11 @@ <h2 class="headline">{% trans "Comment flagged" %}</h2>
                 </div>
 
 				<ul class="comment-actions">
-					<li><a href="#">{% trans "delete" %}</a></li>
+                    {% if not flag.comment.is_removed %}
+                        <li><a href="{% url "spirit:comment:delete" flag.comment.pk %}" target="_blank">{% trans "delete" %}</a></li>
+                    {% else %}
+                        <li><a href="{% url "spirit:comment:undelete" flag.comment.pk %}" target="_blank">{% trans "undelete" %}</a></li>
```

Comment:
Why the `target="_blank"`?

Reply:
Because, back to the admin page is deep. Post and refresh current page would be better.

Reply:
Ah, I see. Adding `class="js-post"` and `?next=...` should take care of that (just like the `logout` does)